src/main/java/teho/main/Ui.java

```diff
@@ -0,0 +1,145 @@
+package teho.main;
+import java.lang.StringBuilder;
+/**
+ * Deals with interactions with the user through generating messages.
+ */
+
+public class Ui {
+    /**
+     * Generates hello message when TehO chatbot starts.
+     *
+     * @return String representation of hello message
+     */
+    public static String generateHelloMessage() {
```

Comment:
I like the naming of the methods in this class, they are consistent and clear to the reader.